_includes/tutorials/session-windows/kstreams/code/src/main/java/io/confluent/developer/SessionWindow.java

```diff
@@ -83,43 +69,39 @@ public Topology buildTopology(Properties envProps) {
     }
 
 
-    static <T extends SpecificRecord> SpecificAvroSerde<T> getSpecificAvroSerde(final Properties envProps) {
+    static <T extends SpecificRecord> SpecificAvroSerde<T> getSpecificAvroSerde(final Properties allProps) {
         final SpecificAvroSerde<T> specificAvroSerde = new SpecificAvroSerde<>();
-
-        final HashMap<String, String> serdeConfig = new HashMap<>();
+        final Map<String, String> serdeConfig = (Map)allProps;
         serdeConfig.put(AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG,
-                envProps.getProperty("schema.registry.url"));
-
+                allProps.getProperty("schema.registry.url"));
```

Comment:
Do we need lines 75 and 76 now? you pass in all the properties on line 77.  Doesn't hurt anything just an unnecessary step

Reply:
Yea totally right. I updated the code accordingly in commit https://github.com/confluentinc/kafka-tutorials/pull/837/commits/461638472275d214eb8e282891ae49aae9cff0ac and verified the test passes in semaphore.